xml/art_solution_monitoring.xml

```diff
@@ -0,0 +1,249 @@
+      on how to monitor and troubleshoot clusters.
+    </para>
+    <para>
+      The exporters showed here can be combined and integrated with Prometheus
+      and Grafana to produce complex dashboards.
+    </para>
+    <para>
+      &suse; supports Prometheus and Grafana through &suma; 4.0. Some Grafana
+      dashboards for &hana;, &s4h;, &netweaver;, and the cluster monitoring
+      are provided by &suse; via Grafana community dashboards.
+    </para>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-terms">
+    <title>Terminology</title>
+    <para></para>
+    <variablelist>
+      <varlistentry>
+        <term>Grafana</term>
+        <listitem>
+          <para>
+            An interactive, visualization and analytics web application.
+            With dashboards, it provides methods to visualize, explore,
+            alert on, and query your metrics.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Prometheus</term>
+        <listitem>
+          <para>
+            A systems monitoring and alerting toolkit. It collects and evaluates
+            metrics, displays the result, and trigger possible alerts
+            when a observed condition is true.
+            Metrics can be collected from different targets at given intervals.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-hana-db-exp">
+    <title>&hana; Database Exporter</title>
+    <para>
+      Export &hana; database metrics.
+      The exporter is able to export the
+      metrics from more than one database or tenant. This option
+      (<option>multi_tenant</option>) is enabled by default.
+    </para>
+    <formalpara>
+      <title>Package:</title>
+      <para><package>prometheus-hanadb_exporter</package></para>
+    </formalpara>
+    <formalpara>
+      <title>Link:</title>
+      <para>
+        See <link xlink:href="https://github.com/SUSE/hanadb_exporter"/>.
+      </para>
+    </formalpara>
+    <itemizedlist>
+      <title>Export Metrics in Prometheus Format</title>
+      <listitem>
+        <para>
+          Memory metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          CPU metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Disk usage metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          I/O metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Network metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Top queries consuming time and memory
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          System replication metrics
+        </para>
+      </listitem>
+    </itemizedlist>
+    <remark>toms 2020-07-02: Regarding Installation: Do we need really need to
```

Comment:
The package can be installed using `zypper`. The package is available from SLE12-SP3 to SLE15-SP2.
The configuration and execution is the same though (set the configuration files properly and start the daemon).

The installation and configuration can be done using `saphanabootstrap-formula` too.

Reply:
Ok, good. :+1:  I guess, it is again the `network:ha-clustering:sap-deployments:devel` repo to use, right? 

> The configuration and execution is the same though (set the configuration files properly and start the daemon).

Just to be on the same page, what exactly do you mean with that?

> The installation and configuration can be done using saphanabootstrap-formula too.

Ok, if I understood you correct, I collected your above information and put that in the right order and this is the result:

1. Install and configure your machine with the `saphanabootstrap-formula`.
2. Install the package `prometheus-hanadb_exporter`.
3. Set the configuration files (where?)
4. Start the daemon (name?)

Is that correct? :wink:

Reply:
By parts XD:

1. Yes, `network:ha-clustering:sap-deployments:devel` is our development repository. Anyway, all of these packages (formulas, involved packages, exporters) are available (or will be, some packages submission is still in progress) in SLES4SAP from SLE12 SP3 to SLE15 SP2 

2. The saphanabootstrap-formula can install and configure the `hanadb_exporter`. This is one of its features (basically the formulas runs `zypper`, moves the configuration files to the correct locations and starts the daemon)

3. The last part, about the steps.
`saphanabootstrap-formula` is not required to have the exporter running, it is just one one to install and configure it. The manual way would be:
- Install prometheus-hanadb_exporter - `zypper in prometheus-hanadb_exporter`
- Install one of the SAP HANA python apis (pyhdb package (open source pacakge, we don't ship it by now, or dbapi, python package that is stored in the HANA client)
- Create a configuration file (example in https://github.com/SUSE/hanadb_exporter/blob/master/config.json.example)
- Copy the new config file and the metrics file to the correct locations ('/etc/hanadb_exporter`)
- Start the daemon

Eveything is explained here: https://github.com/SUSE/hanadb_exporter
Maybe we should avoid adding the steps in this document and just point to the github repo (what you were doing at the beginning).
As I said, the README file needs some improvements, and we will work on that in the next days

Reply:
> [...] Maybe we should avoid adding the steps in this document and just point to the github repo (what you were doing at the beginning).

Absolutely agree. I've removed the remark and added a short sentence:

    Find installation instruction in the previous link.

(the link is already there, before the sentence).